tutorials/R/Hyperspectral/Intro-hyperspectral/Work-With-Hyperspectral-Data-In-R/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -381,7 +391,7 @@ still not great.
 Let's look at the distribution of reflectance values in 
 our data to figure out what is going on.
 
-```{r hist-data }
+```{r hist-data, fig.cap=c("Histogram of reflectance values for band 9. Reflectance values are on the x-axis, and the frequency is on the y-axis.","Histogram of reflectance values between 0 and 5000 for band 9. Reflectance values are on the x-axis, and the frequency is on the y-axis.","Histogram of reflectance values between 5000 and 15000 for band 9. Reflectance values are on the x-axis, and the frequency is on the y-axis.")}
```

Comment:
Would you please add a description of the histograms? What do these distributions say to you? And how could you convey that take-away point to another person who can't see them?